### PR TITLE
made delete & put, changed post & Config

### PR DESCRIPTION
**Pull request by Jaivardhan1**
can someone please check delete & put, as well as changes made in post & Config

### CODE REVIEW

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/delete.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import { baseURL } from '../../Constants/Config'
+// Universal delete request using axios
+export default function universalDelete(setResponse, endpoint, onError, onSuccess) {
+    console.log('Delete started');
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/delete.js]: delete needs request body as well

Comment (Jaivardhan1) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/delete.js]: done

**Comment by Enoch-Hsiao** [comment]
Can you move get.js, put.js, delete.js, and post.js into a directory (universalHTTPRequests or something of that nature), and also move the constants directory outside of the pages directory (probably in the src directory)

**Comment by Jaivardhan1** [comment]
> Can you move get.js, put.js, delete.js, and post.js into a directory (universalHTTPRequests or something of that nature), and also move the constants directory outside of the pages directory (probably in the src directory)

done

**Review by Enoch-Hsiao — APPROVED** [review]
Looks good to me

**Pull request merged**